packages/react-day-picker/src/components/Table/Table.test.tsx

```diff
@@ -0,0 +1,43 @@
+import * as React from 'react';
+
+import { enGB, enUS } from 'date-fns/locale';
+import { customRender, PageObjects } from 'test';
+
+import { Table } from './Table';
+
+const po = new PageObjects(new Date());
```

Comment:
We need to freeze date with timekeeper: check out how the test is implemented here: https://github.com/gpbl/react-day-picker/blob/95ec208ccbf97f4c440a196c24c809219d5d7bd3/packages/react-day-picker/src/components/Caption/Caption.test.tsx#L17-L18

Reply:
Good to know. I ended up moving the tests to getWeeks.test.ts where they got much simpler -- no react necessary!